src/Dependency/DependencyLoader.php

```diff
@@ -0,0 +1,147 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Dependency;
+
+use App\Console\Executor\DatabaseHyphenationExecutor;
+use App\Console\Executor\EmptyExecutor;
+use App\Console\Executor\FileHyphenationExecutor;
+use App\Console\Executor\HyphenateNotHyphenatedExecutor;
+use App\Console\Executor\LoaderExecutor;
+use App\Console\Hyphenator\ArrayHyphenator;
+use App\Console\Hyphenator\HyphenationRule;
+use App\Console\Loader\RuleFileLoader;
+use App\Console\Loader\WordFileLoader;
+use App\Console\Processor\DatabaseProcessor;
+use App\DB\ConnectionManager;
+use App\DB\QueryBuilder;
+use App\DB\QueryWriterFactory;
+use App\IOUtils;
+use App\Logger\SimpleLogger;
+use App\Model\Rule;
+use App\Paths;
+use App\Repository\MatchRepository;
+use App\Repository\RuleRepository;
+use App\Repository\WordRepository;
+
+class DependencyLoader
```

Comment:
Ensuring this class is not extendable (by making it final) would make it more robust